src/d3d9/d3d9_device.cpp

```diff
@@ -4102,12 +4102,15 @@ namespace dxvk {
 
     const Rc<DxvkBuffer> mappedBuffer = pResource->GetBuffer(Subresource);
 
-    auto formatInfo = imageFormatInfo(pResource->GetFormatMapping().FormatColor);
+    DxvkFormatInfo formatInfo = *imageFormatInfo(pResource->GetFormatMapping().FormatColor);
```

Comment:
Check if the format mapping is valid instead with `IsValid()`. See `D3D9CommonTexture::NormalizeTextureProperties`.